frontend/src/metabase/dashboard/components/AutoApplyFiltersToast/AutoApplyFiltersToast.tsx

```diff
@@ -64,6 +67,8 @@ export default function AutoApplyFilterToast() {
 
       dispatch(
         addUndo({
+          id: autoApplyFiltersToastId,
+          timeout: false,
```

Comment:
by allow specifying a toast ID, we can then programmatically close such toasts with `dismissUndo`